src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/baselines/licenses/Licenses.arcade.json

```diff
@@ -0,0 +1,20 @@
+{
+  "files": [
+    {
+      "path": "Documentation/UnifiedBuild/Foundational-Concepts.md",
+      "detected_license_expression": "proprietary-license"
```

Comment:
Nit: this should be a false positive, no?

Reply:
Fixed in https://github.com/dotnet/installer/pull/17442/commits/e0590e795e648693e47c7a6216ffae8a1457804f, plus some more updates